sdk/exportentities/workflow.go

```diff
@@ -414,15 +374,34 @@ nextDep:
 }
 
 //GetWorkflow returns a fresh sdk.Workflow
-func (w Workflow) GetWorkflow() (*sdk.Workflow, error) {
+func (w Workflow) GetWorkflow(proj *sdk.Project, hookModels []sdk.WorkflowHookModel, outgoingHookModel []sdk.WorkflowHookModel) (*sdk.Workflow, error) {
```

Comment:
I understand why you need the models, but i think it's not fine

If you want to parse a workflow with a project and the workflow relies on some shared data coming from the database you have to handle it in the lower layers of the api, not in the parser

Reply:
I know but like below I wanted to avoid useless field in structure just for asCode.

Reply:
i don't get your point